### PR TITLE
Add circular buffer exercise

### PR DESCRIPTION
**Pull request by Bajger**
Added circular buffer exercise including solution.

### CODE REVIEW

**Comment by Bajger** [comment]
@glennj I have some trouble with configlet lint, with UUID - obviously Pharo generator (by: `UUIDGenerator next`) does not produce in format that is needed for linter. Will find out, how it can be fixed (worst case I will overwrite it manually).
Other than that, you can review tests and solution for circular buffer. Hope it is Pharo idiomatic way.

File: dev/src/Exercise@CircularBuffer/CircularBuffer.class.st [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+]
+
+{ #category : #operations }
+CircularBuffer >> overwrite: aValue [
+
+	^ self bufferFull
+		  ifTrue: [
+			  self buffer removeFirst.
+			  self buffer addLast: aValue ]
+		  ifFalse: [ self write: aValue ]
```
Comment (glennj) [inline dev/src/Exercise@CircularBuffer/CircularBuffer.class.st:64]: re-implementing `write` in here. How about
```smalltalk
self bufferFull ifTrue: [ self buffer removeFirst ].
^ self write: aValue
```

**Review by glennj — APPROVED** [review]

**Comment by Bajger** [comment]
CI green! @ErikSchierboom can approve now.

**Review by glennj — APPROVED** [review]
Looks great!

**Review by ErikSchierboom — APPROVED** [review]
Great!

**Pull request merged**